src/sections/Spotllight.astro

```diff
@@ -62,6 +62,7 @@ const allSpotlights = await getCollection("spotlight");
   .events__grid {
     display: grid;
     gap: var(--lh);
+    grid-template-columns: repeat(1, minmax(0, 1fr));
```

Comment:
There should be a cleaner solution for this one. This is a single column, so the default should just work. I bet there is something on the parent/children that makes it misbehave. Let me spend a second on this one.

Reply:
It was this one interfering and messing up the browser layout recalculation all the time.

```
.spotlight__item-role {
    opacity: 0.5;
    font-style: italic;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  }
```

The easiest solution was to simply force each of the items to recalculate width using `min-width: 0`, a terrible hack. I changed it and also aligned some namings with the page name. 

Your solution was great Preston, but I knew that there si sometihng else that messes things up. I am ging to close this one.